src/core_plugins/i18n/server/tool/i18n_verify_keys.js

```diff
@@ -0,0 +1,99 @@
+import fs from 'fs';
+import path from 'path';
+import Promise from 'bluebird';
+import globProm from 'glob-promise';
```

Comment:
Ok, I am a bit on the fence with this one.  The 'glob-promise' is a handy package in this scenario. Not sure about the use of grunt here. Maybe we chat further on IRC on this?